apps/tx-builder/src/pages/ReviewAndConfirm.tsx

```diff
@@ -247,8 +261,7 @@ const SimulationContainer = styled(Card)`
 
 const Wrapper = styled.main`
   && {
-    padding: 48px;
-    padding-top: 120px;
+    padding: 120px 48px 48px;
```

Comment:
I'm used to this syntax. No problem from my side. I think I even prefer because in a long set of CSS rules you could end up having padding overwriting each others